### PR TITLE
feat: request latest complated aggregated time series with version

### PR DESCRIPTION
**Pull request by MadsDue**
## TODO

Extended request time series to ensure we only get calculation result  from completed calculation.

In a following PR we are resolving the latest completed calculations in order to get the latest version number on the calculation results for the request. 

Please complete and remove this TODO section before entering PR ready for review state.

- [x] Title: Follow [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Write tests
- [ ] ~~Update documentation~~
- [ ] ~~Update~~ [event catalog](https://energinet.atlassian.net/wiki/spaces/D3/pages/555581556/Event+catalog) if there are relevant changes to integration events
- [ ] ~~Inform team Outlaws if there are relevant C4 changes~~

### CODE REVIEW

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/AggregatedTimeSeries.cs [hunk 1]
```diff
@@ -40,4 +42,6 @@ public class AggregatedTimeSeries
     public TimeSeriesType TimeSeriesType { get; init; }
 
     public ProcessType ProcessType { get; init; }
+
+    public Guid CalculationId { get; init; }
```
Comment (JesperGomes) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/AggregatedTimeSeries.cs:46]: Why is this needed. I was hoping that one of the main differences between EnergyResult and AggregatedTimeSeries is that AggregatedTimeSeries is not associated to a specific calculation. I assume that AggregatedTimeSeries can span accross calculation periods?

Comment (MadsDue) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Interfaces/CalculationResults/Model/EnergyResults/AggregatedTimeSeries.cs:46]: As I understands it. 

The AggregatedTimeSeries is a result of a calulation. 

Next step is to find the latest AggregatedTimeSeries for a requested period which is done by finding the latest calculations and remove overlapping AggregatedTimeSerie and then add the calculation version back to the AggregatedTimeSeries. 

But let us take a talk about this :)

**Review by JesperGomes — APPROVED** [review]
